.github/workflows/drupal.yml

```diff
@@ -0,0 +1,71 @@
+name: Drupal
+
+on: [push, pull_request]
```

Comment:
Do we need anything other than push event?

Reply:
Dunno, I'll change for push now.